app/forms/waste_carriers_engine/main_people_form.rb

```diff
@@ -6,29 +6,29 @@ class MainPeopleForm < PersonForm
 
     attr_accessor :business_type
 
+    validates_with MainPersonValidator
+
     def initialize(transient_registration)
       super
       # We only use this for the correct microcopy
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      # We only use this for the correct microcopy
```